claim/gql_mutations.py

```diff
@@ -809,12 +833,55 @@ def async_mutate(cls, user, **data):
                 if item['status'] == ClaimItem.STATUS_PASSED:
                     all_rejected = False
             services = data.pop('services') if 'services' in data else []
+            claimed = 0
+            ClaimServiceElts = []
             for service in services:
                 service_id = service.pop('id')
+                service_linked = service.pop('serviceItemSet', [])
+                logger.debug("service_linked ", service_linked)
+                serviceserviceSet = service.pop('serviceserviceSet', [])
+                logger.debug("serviceserviceSet ", serviceserviceSet)
                 claim.services.filter(id=service_id).update(**service)
+                if ClaimConfig.native_code_for_services == False:
```

Comment:
`if not ClaimConfig.native_code_for_services:` should work in the same way

Reply:
what do you mean

Reply:
`if ClaimConfig.native_code_for_services == False:` -> `if not ClaimConfig.native_code_for_services:`
By this @delcroip . They should have coded this condition in that way I proposed.

Reply:
I did not see the "not" ...